go.mod

```diff
@@ -0,0 +1,12 @@
+module github.com/nikandfor/mq-golang-jms20
+
+go 1.11
+
+require (
+	github.com/davecgh/go-spew v1.1.1
+	github.com/ibm-messaging/mq-golang v3.0.0+incompatible
```

Comment:
To match the existing dep approach can we make the `mq-golang` dependency the latest v4.1.0 please?

Reply:
I've imported by commit hash for now.
To make it available by version number target repo should follow this guide https://github.com/golang/go/wiki/Modules#releasing-modules-v2-or-higher